### PR TITLE
Add Tutorials to Readme

### PR DESCRIPTION
**Pull request by vihar**
> Pull Request Template
>
> Use this template to quickly create a well written pull request. Delete all quotes before creating the pull request.

Fixes # (issue)
> if no issue exists, please create an issue and ask the maintainers about this first

## Type of change

> Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)
- Breaking change (fix or feature that would cause existing functionality to not work as expected)
- This change requires a documentation update

## How Has This Been Tested?

> Please describe the tests that you ran to verify your changes. Provide instructions, so we can reproduce. 
> Please also list any relevant details for your test configuration.

- Test A
- Test B

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by akshayrangasaid** [comment]
We'll need to add to the tutorials. Can you please look at confidence videos and put them? Appsmith needs to be front and center of the tutorials. Build Admin Panel without CSS and HTML is not front and center. They just use appsmith. 

For example:

https://www.youtube.com/watch?v=CYdeJcD4I8A -> Title in the commit is wrong.

https://www.youtube.com/watch?v=eYYYfuW-kEE -> Another example, this one mark as community

https://dev.to/pjmantoss/how-to-build-an-issue-tracker-with-appsmith-204e -> Another example. Mark it as community

Then the video of Nikhil should work along with the mongodb one

**Review by akshayrangasaid — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -40,50 +41,63 @@ Already familiar with Appsmith? [Quickly start building on your own](#%EF%B8%8F-
...
 The following steps introduce you to building a simple user-list dashboard on Appsmith.
+
 1. [Sign up on Appsmith Cloud](https://bit.ly/appsmith-signup-github) or [Deploy Appsmith](https://docs.appsmith.com/setup).
 2. Create a new app within the organization that has already been created for you.
 3. Click on the `+` icon next to the `Queries` section to add a new query in the mock database
-    1. Name the query `usersQuery`.
-    2. Write the query `select * from users limit 5;`.
-    3. Run the query.
+   1. Name the query `usersQuery`.
+   2. Write the query `select * from users limit 5;`.
```
Comment (Nikhil-Nandagopal) [inline README.md:70]: This is a bug without the spaces, it won't show as sub steps when it renders

File: README.md [hunk 2]
```diff
@@ -40,50 +41,63 @@ Already familiar with Appsmith? [Quickly start building on your own](#%EF%B8%8F-
...
 6. Hit the Deploy button and checkout the view mode of the app.
 
-Congratulations 🎉 You just built your first app on Appsmith! 
+Congratulations 🎉 You just built your first app on Appsmith!
 Connect your own data to build apps for your team. [Read more here.](https://docs.appsmith.com/core-concepts)
 
+## 📚 Tutorials
+
+1. [Building an Admin Panel on MongoDB using Appsmith](https://blog.appsmith.com/building-an-admin-panel-with-mongodb-using-appsmith) ([Video](https://www.youtube.com/watch?v=tisUaIgI86k))
+2. [Building a customer support dashboard in Appsmith](https://www.youtube.com/watch?v=-O_6OLREEzo&t=272s)
```
Comment (Nikhil-Nandagopal) [inline README.md:82]: can we merge this with the demo section?

**Review by Nikhil-Nandagopal — CHANGES_REQUESTED** [review]
Just had one suggestion

File: README.md (hunk 2 above)
Comment (vihar) [reply README.md:82]: Yes, we could merge this and add a tag as `Demo`. @akshayrangasaid thoughts?

Comment (akshayrangasaid) [reply README.md:82]: @Nikhil-Nandagopal Right now the demo apps and tutorials do not have the same flow, we'll either convert demo to tutorials or make them easier to grok and merge. Think for now, keeping them separate is best to maintain consistency.

Comment (Nikhil-Nandagopal) [reply README.md:82]: @akshayrangasaid what if we kill the demo apps, show only tutorials but have a link to a forkable version of the tutorial app at the start of every tutorial?

Comment (akshayrangasaid) [reply README.md:82]: @Nikhil-Nandagopal yep for sure. I like the keywords the demo apps show, thus want to keep them on while we get some content out, but not too fussed to just kill them now only and bring them back with a more solid onboarding/tutorial. 


Longer-term with collab + notes, we can just have forkable apps with notes in it without the overhead of a tut.

**Review by Nikhil-Nandagopal — APPROVED** [review]

**Pull request merged**